docs/monitors/elasticsearch.md

```diff
@@ -11,77 +11,116 @@ This monitor collects cluster level and index level stats only from the current
-pool stats. It is possible to enable enhanced stats for each group separately. Here's an example:
+The monitor collects a subset of node stats of JVM, process, HTTP, transport,
+indices and thread pool stats. It is possible to enable enhanced stats for each
+stat group separately. Here's an example:
 
-```
+```yaml
 monitors:
 - type: elasticsearch
- host: localhost
- port: 9200
- enableEnhancedHTTPStats: true
- enableEnhancedJVMStats: true
- enableEnhancedProcessStats: true
- enableEnhancedThreadPoolStats: true
- enableEnhancedTransportStats: true
- enableEnhancedNodeIndicesStats:
-	- indexing
-	- warmer
- - get
+  host: localhost
+  port: 9200
+  enableEnhancedHTTPStats: true
+  enableEnhancedJVMStats: true
+  enableEnhancedProcessStats: true
+  enableEnhancedThreadPoolStats: true
+  enableEnhancedTransportStats: true
+  enableEnhancedNodeIndicesStats:
+   - indexing
+   - warmer
+   - get
+
 ```
 
-The `enableEnhancedNodeIndicesStats` option takes a list of index stats groups for which enhanced
-stats will be collected. A comprehensive list of all available such groups can be found
-[here] (https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html#node-indices-stats).
+The `enableEnhancedNodeIndicesStats` option takes a list of index stats groups
+for which enhanced stats will be collected. A comprehensive list of all
+such available groups can be found [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html#node-indices-stats).
 
-Note that the `enableEnhancedIndexStatsForIndexGroups` is similar to `enableEnhancedNodeIndicesStats`,
-but for index level stats
+Note that the `enableEnhancedIndexStatsForIndexGroups` is similar to
+`enableEnhancedNodeIndicesStats`, but for index level stats.
 
-By default the monitor collects a subset of index stats of total aggregation type (see docs for details).
-It is possible to enable index stats of primaries aggregation type too. Total for an index stat
-aggregates across all shards. Whereas, Primaries only reflect the stats from primary shards. An example
-configuration to enable index stats from Primary shards too
+## Thread Pools
 
-```
+By default thread pool stats from the "search" and "index" thread pools are collected. To collect
+stats from other thread pools specify the `threadPools` config option:
+
+```yaml
 monitors:
 - type: elasticsearch
- host: localhost
- port: 9200
- enableIndexStatsPrimaries: true
+  host: localhost
+  port: 9200
+  threadPools:
+  - bulk
+  - warmer
+  - listener
 ```
 
-It is possible to collect index level stats that are aggregated across all indexes, rather than one a per index level
+Here is a list of valid thread pools by Elasticsearch version:
+
+| thread pool name | ES 1.x | ES 2.0 | ES 2.1+ |
+|------------------|--------|--------|--------|
+| merge            | &#x2713;      |        |        |
+| optimize         |&#x2713;     |        |        |
```

Comment:
Is there a reason this can't be the actual unicode value '✓' instead of as an escape sequence?

Reply:
I don't really know enough about the process that it goes through from the markdown file to the web UI to say.  This table was just copied from the integrations repo as is so I think it is safer to leave as it was.

Reply:
I think I changed this to use escape sequence so it's compatible with both the app and read the docs.

Reply:
Don't know why the app wouldn't be able to consume unicode. Docs handles it fine though.